nvdaHelper/remote/winword.cpp

```diff
@@ -711,7 +712,9 @@ inline int generateInlineShapeXML(IDispatch* pDispatchRange, int offset, wostrin
 		SysFreeString(altText);
 	}
 	altText=NULL;
-	XMLStream<<L"<control _startOfNode=\"1\" role=\""<<((shapeType==wdInlineShapePicture||shapeType==wdInlineShapeLinkedPicture)?L"graphic":(shapeType==wdInlineShapeChart?L"chart":L"object"))<<L"\" value=\""<<altTextStr<<L"\"";
+	XMLStream<<L"<control _startOfNode=\"1\" ";
+	XMLStream<<L"role=\""<<((shapeType==wdInlineShapePicture||shapeType==wdInlineShapeLinkedPicture)?L"graphic":(shapeType==wdInlineShapeChart?L"chart":L"object"))<<L"\" ";

```

Comment:
Please pull the logic out of this. Then it is easier to determine the structure of the xml separate from the flow deciding values.

```suggestion
	auto roleText = L"object";
	if(shapeType==wdInlineShapePicture||shapeType==wdInlineShapeLinkedPicture){
		roleText=L"graphic";
	}
	if(shapeType==wdInlineShapeChart){
		roleText=L"chart";
	}
	
	XMLStream<<L"role=\""<<roleText<<L"\" ";
```